### DISCUSSION TITLE
Add Toolbar Menu

### DISCUSSION THREAD

**Discussion by naurahsopian** [cyanfish/naps2#237] [Development]
How do I add a menu button to the naps2 toolbar?

**Comment by cyanfish** [comment]
See DesktopForm:
https://github.com/cyanfish/naps2/blob/a82f5671e685ccfdd01cf6ff2a0d359ff545d9d3/NAPS2.Lib/EtoForms/Ui/DesktopForm.cs#L241

If you want it to be cross-platform, the Mac interface has a different setup:
https://github.com/cyanfish/naps2/blob/a82f5671e685ccfdd01cf6ff2a0d359ff545d9d3/NAPS2.Lib.Mac/EtoForms/Ui/MacDesktopForm.cs#L57
